routing/router.go

```diff
@@ -433,6 +421,30 @@ func (r *ChannelRouter) Start() error {
 		}
 	}
 
+	// If AssumeChannelValid is present, then we won't rely on pruning
+	// channels from the graph based on their spentness, but whether they
+	// are considered zombies or not.
+	if r.cfg.AssumeChannelValid {
+		if err := r.pruneZombieChans(); err != nil {
```

Comment:
This skips calling `PruneGraphNodes`.

Reply:
Will be including these changes in another PR.